src/geo_div/geo_div.cpp

```diff
@@ -145,14 +175,20 @@ std::vector<Polygon_with_holes> GeoDiv::polygons_with_holes() const
   return polygons_with_holes_;
 }
 
-void GeoDiv::push_back(const Polygon_with_holes &pwh)
+void GeoDiv::push_back_ellipse(const Ellipse ell)
+{
+  min_ellipses_.push_back(ell);
+  return;
```

Comment:
Clang-Tidy: Redundant return statement at the end of a function with a void return type